Lombiq.Vsix.Orchard/Services/LogWatcher/BlinkStickManager.cs

```diff
@@ -9,11 +9,9 @@ namespace Lombiq.Vsix.Orchard.Services.LogWatcher
     public sealed class BlinkStickManager : IBlinkStickManager
     {
         private readonly object _lock = new object();
-        private BlinkStick _blinkStick = null;
-        private bool _isInitialized = false;
-        private Task _backgroundTask = null;
```

Comment:
Fura lenne, ha előbb-utóbb nem takarítja fel a GC ezeket a taskokat így hogy nincs hozzájuk referencia. Szóval ezt megtartanám.

Reply:
Van referencia a Task-ra, de nem osztály szintű változó:
![image](https://user-images.githubusercontent.com/18496537/142758347-de04f70f-3be2-4b4c-9647-0f0b7d58da26.png)

Reply:
Az egy local változó, ami a `Blink()` végén megszűnik (illetve nem vagyok benne biztos, hogy működik a discard e tekintetben, lehet még rövidebb életű). Az, hogy Debug mode-ban localban ez működik, az eltűnő referencia miatt egyáltalán nem biztos, hogy Release mode-ban később a usereinknél működni fog, órákon keresztül is.

Az eredeti amúgy objektum (azaz példány)-szintű, nem osztály-szintű (azaz static) volt, előbbire van szükség.